libs/wingsdk/src/core/app.ts

```diff
@@ -96,8 +96,8 @@ export abstract class App extends Construct {
    * Returns the root app.
    */
   public static of(scope: Construct): App {
-    if (scope instanceof App) {
-      return scope;
+    if ((scope as any)._isApp) {
+      return scope as any;
```

Comment:
The `instanceof` was misbehaving with external platform App instances which load in their own `@winglang/sdk`. I wrestled around with sandboxing but could not get things reasonably working.

Happy for other suggestions